schemas.py

```diff
@@ -0,0 +1,36 @@
+from pydantic import BaseModel
+from datetime import date
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class Author(AuthorBase):
+    id: int
+
+    class Config:
+        orm_mode = True
```

Comment:
```suggestion
        from_attributes = True
```
you can google, why I suggested it )